### PR TITLE
CSP-694 Rename IsActive to IsAttending

### PR DESCRIPTION
**Pull request by SawNawDfE**

### CODE REVIEW

File: src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.refactorlog [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Operations Version="1.0" xmlns="http://schemas.microsoft.com/sqlserver/dac/Serialization/2012/02">
+  <Operation Name="Rename Refactor" Key="ac5c46e1-c691-4e37-88e1-c3abf90980b8" ChangeDateTime="06/12/2023 16:52:59">
+    <Property Name="ElementName" Value="[dbo].[Attendance].[IsActive]" />
+    <Property Name="ElementType" Value="SqlSimpleColumn" />
+    <Property Name="ParentElementName" Value="[dbo].[Attendance]" />
+    <Property Name="ParentElementType" Value="SqlTable" />
+    <Property Name="NewName" Value="[IsAttending]" />
+  </Operation>
+</Operations>
```
Comment (MarkFCain) [inline src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.refactorlog:10]: I don't think we need this in the codebase.  Maybe add *.refactorlog to your .gitignore?

File: src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.sqlproj [hunk 2]
```diff
@@ -92,4 +92,7 @@
   <ItemGroup>
     <PreDeploy Include="Scripts\Script.PreDeployment1.sql" />
   </ItemGroup>
+  <ItemGroup>
+    <RefactorLog Include="SFA.DAS.AANHub.Database.refactorlog" />
+  </ItemGroup>
 </Project>
```
Comment (MarkFCain) [inline src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.sqlproj:98]: and remove this too

**Review by MarkFCain — DISMISSED** [review]

**Comment by SawNawDfE** [comment]
Closed as not ready for review

File: src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.sqlproj (hunk 2 above)
Comment (SawNawDfE) [reply src/SFA.DAS.AANHub.Database/SFA.DAS.AANHub.Database.sqlproj]: REmoved

**Review by MarkFCain — APPROVED** [review]

**Review by DevenKShah — DISMISSED** [review]

**Review by MarkFCain — DISMISSED** [review]

**Review by MarkFCain — APPROVED** [review]

**Pull request merged**